packages/ncursesw.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Ncursesw < Package
```

Comment:
Would we need to add this new package as a dependency for other packages or should it be a core package?

Reply:
Core package.

Reply:
Although the different libraries are source compatible, they aren't binary compatible and anything built with ncursesw will break with its removal

Reply:
In order to make it a core package, we would need to do the following:

1. Add to install.sh -or- add as a dependency of another core package so it is "inherited"
2. Add to tools/core_packages.txt